server/Migrations/20221222215900_done.cs

```diff
@@ -0,0 +1,387 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Add the whole migrations folder to the gitignore pls @GeneDrive

Reply:
reeeeeee

Reply:
I did just now

Reply:
This file shouldn't exist after the gitignore thing. Find a way to ignore it